### PR TITLE
Suuport for Convert_Timezone with two args for Snowflake

### PR DESCRIPTION
**Pull request by Rajeshwari4**

### CODE REVIEW

File: core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java [hunk 1]
```diff
@@ -8756,6 +8756,40 @@ private void checkLiteral2(String expression, String expected) {
...
+        builder.call(SqlLibraryOperators.PARSE_TIMESTAMP_WITH_TIMEZONE,
+        builder.literal("%c%z"), builder.call(SqlLibraryOperators.FORMAT_TIMESTAMP,
+            builder.literal("%c%z"),
+            builder.cast(builder.literal("2008-08-21 07:23:54"), SqlTypeName.TIMESTAMP),
+            builder.literal("America/Los_Angeles")));
+    final RelNode root = builder
+        .scan("EMP")
+        .project(builder.alias(parseTSNode, "timestamp"))
+        .build();
+    final String expectedBiqQuery =
```
Comment (nosheenSheikh02) [inline core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java:8785]: make this expectedBigQuery

File: core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java [hunk 2]
```diff
@@ -10726,6 +10760,19 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
     assertThat(toSql(root, DatabaseProduct.SPARK.getDialect()), isLinux(expSprk));
   }
 
+  @Test public void testTimeWithTimezoneFunction() {
+    final RelBuilder builder = relBuilder();
+    final RexNode formatTimestampRexNode2 = builder.call(SqlLibraryOperators.FORMAT_TIMESTAMP,
```
Comment (nosheenSheikh02) [inline core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java:10765]: make it `formatTimestampRexNode`

File: core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java [hunk 3]
```diff
@@ -10726,6 +10760,19 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
...
+        builder.literal("%c%z"), builder.call(SqlLibraryOperators.CURRENT_TIMESTAMP));
+    final RelNode root = builder
+        .scan("EMP")
+        .project(builder.alias(formatTimestampRexNode2, "FD2"))
+        .build();
+    final String expectedBiqQuery = "SELECT FORMAT_TIMESTAMP('%c%z', CURRENT_DATETIME()) AS FD2\n"
```
Comment (nosheenSheikh02) [inline core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java:10771]: make this expectedBigQuery

**Review by nosheenSheikh02 — CHANGES_REQUESTED** [review]

**Review by nosheenSheikh02 — APPROVED** [review]

**Review by dm-shantanuc — APPROVED** [review]
LGTM

**Pull request merged**